support-frontend/assets/components/svgs/email.jsx

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import React from 'react';
+
+// A monochrome version of the Twitter logo.
+export default function SvgEmail() {
+  return (
+    <svg
+      className="svg-linkedin"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 30 30"
+      preserveAspectRatio="xMinYMid"
+      aria-labelledby="svg-email-title"
+    >
+      <title id="svg-email-title">Email</title>
```

Comment:
let's either name this correctly or get rid of the title
```suggestion
      <title id="svg-email-title">LinkedIn</title>
```